core/src/game_state.rs

```diff
@@ -158,6 +158,18 @@ impl Default for FirstLandlordSelectionPolicy {
     }
 }
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum JokerOverbidSelectionPolicy {
```

Comment:
perhaps just `BidPolicy`, and the options are `JokerOrGreaterLength` and `GreaterLength`?

Reply:
did this get changed ?